http/src/request/channel/message/update_message.rs

```diff
@@ -10,9 +10,15 @@ use twilight_model::{
 };
 
 #[derive(Clone, Debug)]
+/// The error created when a message can not be updated.
```

Comment:
Again I think this makes it sound like the updating failed, when it failed on the validation step.